LICENSE

```diff
@@ -1,7 +1,7 @@
-Copyright (c) 2016, Los Alamos National Security, LLC
+Copyright (c) 2017, Los Alamos National Security, LLC
```

Comment:
I guess we should do the range as that is consistent with the copyright in the individual files.